File: mobile/src/main/java/treehou/se/habit/ui/servers/create/myopenhab/CreateMyOpenhabComponent.java
package treehou.se.habit.ui.servers.create.myopenhab;


import dagger.Subcomponent;
import treehou.se.habit.module.ActivityScope;
import treehou.se.habit.module.FragmentComponent;
import treehou.se.habit.module.FragmentComponentBuilder;
import treehou.se.habit.ui.settings.SettingsFragment;

@ActivityScope
@Subcomponent(
        modules = CreateMyOpenhabModule.class
)
public interface CreateMyOpenhabComponent extends FragmentComponent<CreateMyOpenhabFragment> {

    @Subcomponent.Builder
    interface Builder extends FragmentComponentBuilder<CreateMyOpenhabModule, CreateMyOpenhabComponent> {
    }
}

File: mobile/src/main/java/treehou/se/habit/dagger/fragment/CreateMyOpenhabComponent.kt
package treehou.se.habit.dagger.fragment


import dagger.Subcomponent
import treehou.se.habit.dagger.FragmentComponent
import treehou.se.habit.dagger.FragmentComponentBuilder
import treehou.se.habit.dagger.scopes.ActivityScope
import treehou.se.habit.ui.servers.create.myopenhab.CreateMyOpenhabFragment

@ActivityScope
@Subcomponent(modules = arrayOf(CreateMyOpenhabModule::class))
interface CreateMyOpenhabComponent : FragmentComponent<CreateMyOpenhabFragment> {

    @Subcomponent.Builder
    interface Builder : FragmentComponentBuilder<CreateMyOpenhabModule, CreateMyOpenhabComponent>
}